### PR TITLE
Add support for Drink Me

### PR DESCRIPTION
**Pull request by yatara-cc**
Add support and keymaps for Drink Me by Yatara

Contains `default`, `via` and several other example keymaps.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yatara/drink_me/keymaps/queen/keymap.c [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    TD_K0,
+    TD_K1,
+    TD_K2,
+    TD_K3
+};
+
+
+void td_spade_lnx (qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 1) {
+        send_unicode_hex_string("0x2660");
```
Comment (drashna) [inline keyboards/yatara/drink_me/keymaps/queen/keymap.c]:
```suggestion
        send_unicode_hex_string("2660");
```

File: keyboards/yatara/drink_me/keymaps/queen/keymap.c [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    } else {
+        set_unicode_input_mode(UC_LNX);
+    }
+    reset_tap_dance(state);
+}
+
+
+void td_diamond_osx (qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 1) {
+        send_unicode_hex_string("0x2666");
```
Comment (drashna) [inline keyboards/yatara/drink_me/keymaps/queen/keymap.c]:
```suggestion
        send_unicode_hex_string("2666");
```

File: keyboards/yatara/drink_me/keymaps/queen/keymap.c [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    } else {
+        set_unicode_input_mode(UC_OSX);
+    }
+    reset_tap_dance(state);
+}
+
+
+void td_club_win (qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 1) {
+        send_unicode_hex_string("0x2663");
```
Comment (drashna) [inline keyboards/yatara/drink_me/keymaps/queen/keymap.c]:
```suggestion
        send_unicode_hex_string("2663");
```

File: keyboards/yatara/drink_me/keymaps/queen/keymap.c [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    } else {
+        set_unicode_input_mode(UC_WIN);
+    }
+    reset_tap_dance(state);
+}
+
+
+void td_heart_winc (qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 1) {
+        send_unicode_hex_string("0x2665");
```
Comment (drashna) [inline keyboards/yatara/drink_me/keymaps/queen/keymap.c]:
```suggestion
        send_unicode_hex_string("2665");
```

File: keyboards/yatara/drink_me/rules.mk [hunk 5]
```diff
@@ -0,0 +1,3 @@
+MCU = atmega32u2
+
+BOOTLOADER = atmel-dfu
```
Comment (noroadsleft) [inline keyboards/yatara/drink_me/rules.mk]:
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output on port C6
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```

**Comment by yatara-cc** [comment]
@noroadsleft @drashna Thank you both for your clear and explanatory suggestions. This is my first time using the GitHub code review process. It's not really clear to me whether I am supposed to click the "re-request review" buttons after responding to suggestions, or perform some other action to say that they've been committed. If one of you could clarify that would be appreciated.

I also wanted to say that I really appreciate all the work that has been put in to this project, both in terms of the software itself and all the support and interaction with the community.

**Comment by noroadsleft** [comment]
> It's not really clear to me whether I am supposed to click the "re-request review" buttons after responding to suggestions, or perform some other action to say that they've been committed. If one of you could clarify that would be appreciated.

You can if you want. I try to regularly check pull requests that I review (once a day at least, if they have updates).

Thanks for the kind words, and your interest in contributing!

**Pull request merged**